platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -186,4 +223,81 @@ - (void)cancel;
     _mbglMapSnapshotter.reset();
 }
 
+- (NSURL*)styleURL;
+{
+    NSString *styleURLString = @(_mbglMapSnapshotter->getStyleURL().c_str());
+    return styleURLString && styleURLString.length > 0 ? [NSURL URLWithString:styleURLString] : nil;
```

Comment:
Can `styleURL` be `nil`? If so, then the property should be declared `nullable` in the interface. Otherwise, we can assert here that the style is non-`nil`.